changelog/8682.misc.md

```diff
@@ -0,0 +1 @@
+Filtering import errors before sending to sentry
```

Comment:
I would format this in active voice, e.g. 'Filters out `ImportError` from Sentry tracking'.

Reply:
updated